Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -668,6 +668,68 @@ private function __fetchModuleData pExtensionFile
    return tDataA
 end __fetchModuleData
 
+private function __extensionIsBuiltin pID
+   set the itemdelimiter to "."
+   return the number of items in pID is 3
+end __extensionIsBuiltin
+
```

Comment:
No, it isn't - I wasn't sure exactly what the conditions were / will be which was why I made it a separate function so we can easily change it.